### DISCUSSION TITLE
The end of tweepy ?

### DISCUSSION THREAD

**Discussion by blepw** [tweepy/tweepy#2100] [Questions] [answered]
Elon Musk disabled all apps that use the twitter Api and is now charging insane amounts of money

**Comment by Harmon758** [comment, marked as answer]
From [the Tweepy Discord server](https://discord.gg/bJvqnhg):

> I'll probably push out a release or two prior to the changes, but otherwise, there's not really any information on if there's any required changes to Tweepy.
> I also probably won't continue to actively develop the library afterwards. Unless I get sponsored $100+ / month on GitHub, specifically for Tweepy, it'll probably be in maintenance mode in terms of my contributions.

\- @Harmon758

https://discord.com/channels/432685901596852224/432685901596852226/1091385913629757542

> I have no plans myself to resume active development. Given recent developments unless there is any fixes @​Harmon or myself approve the library will probably stop receiving active updates. I lost interest in Twitter long ago and recent developments gives me no motivation to contribute to their ecosystem anymore.

\- @joshthecoder

https://discord.com/channels/432685901596852224/432685901596852226/1091433070328037497

Duplicate of #2058 
See also https://twittercommunity.com/t/announcing-new-access-tiers-for-the-twitter-api/188728
